### PR TITLE
Moved cmdlets into runbits.

### PR DESCRIPTION
**Pull request by mitchell-as**
<table>
<td>
  <a href="https://activestatef.atlassian.net/browse/DX-2213" title="DX-2213" target="_blank">TaskDX-2213</a>  Unify runbits and cmdlets
  </td></table>
  <br />

### CODE REVIEW

File: internal/runbits/git/test/integration/git_test.go [hunk 1]
```diff
@@ -79,7 +79,7 @@ func (suite *GitTestSuite) AfterTest(suiteName, testName string) {
 }
 
 func (suite *GitTestSuite) TestEnsureCorrectProject() {
-	err := gitlet.EnsureCorrectProject("test-owner", "test-project", filepath.Join(suite.dir, constants.ConfigFileName), "test-repo", outputhelper.NewCatcher(), blackhole.New())
+	err := runbitsGit.EnsureCorrectProject("test-owner", "test-project", filepath.Join(suite.dir, constants.ConfigFileName), "test-repo", outputhelper.NewCatcher(), blackhole.New())
```
Comment (MDrakos) [inline internal/runbits/git/test/integration/git_test.go:82]: Nit, could probably just use `git` as the package name

**Review by MDrakos — APPROVED** [review]
Nice!

File: internal/runbits/git/test/integration/git_test.go (hunk 1 above)
Comment (mitchell-as) [reply internal/runbits/git/test/integration/git_test.go:82]: I tried, but https://github.com/ActiveState/cli/pull/2866/files#diff-de5d57791df0ffc2779a980eed8972e44d069c7d6bbe4f59a2912b3dfcfd6cf6R13 uses `git` :(

**Comment by mitchell-as** [comment]
Please re-review. I had to fix an upstream merge conflict.

**Review by MDrakos — APPROVED** [review]

**Pull request merged**